Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import './Header.css';
 
+let today = new Date().toLocaleDateString('en-GB', {  
```

Comment:
Great find on this solution to displaying the Date here. There are lots of ways to solve this including using moment.js, but I really like how straightforward this is.